apps/ui/src/composables/useWhiteLabel.ts

```diff
@@ -1,34 +1,58 @@
 import { useQueryClient } from '@tanstack/vue-query';
 import { getNetwork, metadataNetwork } from '@/networks';
-import { Space } from '@/types';
+import { SkinSettings, Space } from '@/types';
 
 const DEFAULT_DOMAIN = import.meta.env.VITE_HOST || 'localhost';
 const domain = window.location.hostname;
 
+// Hardcoded white label mappings for onchain spaces
```

Comment:
```suggestion
// Hardcoded whitelabel mappings for onchain spaces
```

Reply:
We should always use Whitelabel or whitelabel, not WhiteLabel or WHITE_LABEL. A white label is not the same as a a whitelabel.